### PR TITLE
Issue #4043: Split and Organize Checkstyle inputs by Test for OuterTy…

### PR DESCRIPTION
**Pull request by tashish**
Pull request for #4043

### CODE REVIEW

**Comment by tashish** [comment]
@Vladlis @romani I have closed the PR with issues and recreated this PR which has passed all the tests. Kindly review the PR.

**Comment by Vladlis** [comment]
In future please follow points 8, 9 from https://github.com/checkstyle/checkstyle/wiki/PR-rules

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/outertypenumber/InputOuterTypeNumber.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
-package com.puppycrawl.tools.checkstyle.checks.sizes;
+package com.puppycrawl.tools.checkstyle.checks.sizes.outertypenumber;
 
 /**Contains empty inner class for OuterTypeNumberCheckTest*/
-public class InputOuterTypeNumber {
+public final class InputOuterTypeNumber {
```
Comment (Vladlis) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/outertypenumber/InputOuterTypeNumber.java]: Do not change this line

Comment (tashish) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/outertypenumber/InputOuterTypeNumber.java]: Thanks. Added 'final' back to the class declaration. Thanks @Vladlis

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/sizes/OuterTypeNumberCheckTest.java [hunk 2]
```diff
@@ -33,10 +33,16 @@
 import com.puppycrawl.tools.checkstyle.utils.CommonUtils;
 
 public class OuterTypeNumberCheckTest extends BaseCheckTestSupport {
+
+    /**
+     * Prefix for input directory of test case
+     */
+    private static final String TEST_RESOURCE_PATH_PREFIX = "checks" + File.separator
+        + "sizes" + File.separator + "outertypenumber" + File.separator;
+
```
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/sizes/OuterTypeNumberCheckTest.java]: please move it back to method, all other tests have it there, so lets keep the same format.

**Comment by romani** [comment]
@tashish , please squash commits in one.

**Comment by rnveach** [comment]
@tashish ping

**Comment by rnveach** [comment]
@tashish ping
Since we are out of proposal deadline, do you think you can finish this?

**Comment by rnveach** [comment]
@tashish If there is no activity by May 1st, PR will be closed as incomplete and issue will be open to others.

**Comment by romani** [comment]
PRs is closed as we lost connection with student, any contributor could continue this work to make it done.

**Pull request closed without merging**